openapi/rhose-api.yaml

```diff
@@ -939,6 +943,10 @@ components:
           $ref: '#/components/schemas/Source'
         resolvedAction:
           $ref: '#/components/schemas/Action'
+        isErrorHandler:
```

Comment:
By removing the `isErrorHandler` flag on the `ProcessorDefinition` you now leak the properties to the REST API which is not nice as it affects `Processor` creation and updates.

Reply:
I had this issue too and had to settle on the approach I took.